### ISSUE TITLE
[Core] ID extension / Boss respawn

### ISSUE THREAD

**Issue by Plege** [WoWManiaUK/Redemption#1168]
The following issue has been observed at isolated times.

Some Bosses in ICC respawn even when a raid lock is extended. I recall it happened even within the same week with no extension. 

Important to note is that not all the bosses did respawn and that we were still able to teleport to our current progress (eg. LK) but as soon as the encounter started we were teleported to lights hammer because other bosses were still up that shouldn't be.

We were able to go back to those encounters ( Lady Deathwhisper  npc=36855 and Gunship) after completing them we were able to resume.

In short: Lady D and Gunship respawned eventhough they were killed before, other bosses didn't respawn. After killing them the problem was solved.

Regards
Pleg
